### DISCUSSION TITLE
Navigate-to in modal screen

### DISCUSSION THREAD

**Discussion by kmdelucaNG** [Budibase/budibase#7373] [Help] [answered]
Is the _Navigate To_ button action supposed to work from a Modal screen to a non-modal screen?  For example, _Close Modal_ followed by _Navigate To_. From what I can tell it does not.  But if I'm in a full screen or don't do the _Close Modal_, the same _Navigate To_ works fine.

thanks.

**Comment by melohagan** [comment, marked as answer]
Hi @kmdelucaNG 

Thanks for bringing this to our attention. There is an issue raised already: https://github.com/Budibase/budibase/issues/7184
